slab/transfer.go

```diff
@@ -180,16 +248,18 @@ func parallelDownloadSlab(s Slice, hosts []Host) ([][]byte, error) {
 }
 
 // DownloadSlab downloads slab data.
-func DownloadSlab(w io.Writer, s Slice, hosts []Host) error {
```

Comment:
why remove the writer?

Reply:
I wasn't sure whether we could rely on `s.Length` to be correct. I thought it was possible for instance to return a larger length and then the slab movers wouldn't pad the data. So if you don't know the length of the data beforehand, passing a writer doesn't make that much sense.